### PR TITLE
[Australia 2022] - Recap post and links

### PR DESCRIPTION
**Pull request by swapnilogale**
----
:books: Documentation preview :books:: https://writethedocs-www--1864.org.readthedocs.build/en/1864/

### CODE REVIEW

**Comment by swapnilogale** [comment]
Waiting on YouTube links

**Comment by swapnilogale** [comment]
Waiting on Youtube links

File: docs/conf/australia/2022/news/thank-you-recap.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+G'day everyone,
+
+The Write the Docs Australia team is now emerging from the post-conference recovery period, and we wanted to send out a quick message to thank everyone for helping to make our 2022 Australia online conference such a success.
+We appreciate everyone who makes the conference possible: speakers, sponsors, organizers, volunteers, and attendees alike!
+We're happy to share some of the highlights with you.
+
+Conference recap
+=================
+
+We had 95 registrations, and were happy to see folks come back after a pause (on this conference) in 2021. We had the usual mix of main stage and lightning talks, and a social event.
```
Comment (plaindocs) [inline docs/conf/australia/2022/news/thank-you-recap.rst]:
```suggestion
We had almost one hundred attendees and were happy to see folks come back after the Write The Ddocs Australia team took a year out in 2021. We had the usual mix of main stage and lightning talks, and a social event.
```

File: docs/conf/australia/2022/news/thank-you-recap.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+.. _monthly newsletter: http://writethedocs.org/newsletter/
+
+Local Meetups
+-------------
+
+We hope you continue to `build your local or online communities`_ and keep in touch with other documentarians. 
+We have a number of meetups in cities across the country, and if you don’t see your city listed consider `starting`_ a meetup!
+
+New this year are our `Quorum Meetups`_ which are virtual and targeting large geographic areas. There’s already one covering most everyone in the US or in a similar timezone.
```
Comment (plaindocs) [inline docs/conf/australia/2022/news/thank-you-recap.rst]: Do we want all this in here?

File: docs/conf/australia/2022/news/thank-you-recap.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+None of our organizers were approached about any CoC concerns at this event.
+We know that this is, in part, a reflection of the thoughtful and inclusive community that we're so lucky to have.
+
+That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened.
+If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behaviour set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
+
+Ready for more?
+===============
+
+We are already talking 2023 folks! Our `Portland conference <https://www.writethedocs.org/conf/portland/2023/>`__ has announced its dates for 7-9 May and CFPs are now open.
```
Comment (plaindocs) [inline docs/conf/australia/2022/news/thank-you-recap.rst:72]: This too I feel is kinda heavy for an mostly Australian list?

Comment (swapnilogale) [reply docs/conf/australia/2022/news/thank-you-recap.rst:72]: I think this update goes out to everyone WTD, so I'd leave this in, but will take out the Slack/Newsletter/local meetups sections

File: docs/conf/australia/2022/news/thank-you-recap.rst (hunk 2 above)
Comment (swapnilogale) [reply docs/conf/australia/2022/news/thank-you-recap.rst]: Removed it now

**Review by mxsasha — APPROVED** [review]

**Pull request merged**